corgi/tasks/brew.py

```diff
@@ -168,6 +130,70 @@ def slow_fetch_brew_build(
     return build_created or root_created or any_child_created
 
 
+def _get_nested_builds(
+    build_type: str, nested_builds: set[int], force_process: bool, save_product: bool
+) -> None:
+    logger.info("Fetching brew builds for (%s, %s)", nested_builds, build_type)
+    for b_id in nested_builds:
+        logger.info("Requesting fetch of nested build: (%s, %s)", b_id, build_type)
+        slow_fetch_brew_build.delay(
+            b_id, build_type, save_product=save_product, force_process=force_process
+        )
+
+
+def _check_and_load_errata(released_errata_tags, force_process) -> None:
+    """ " for builds with released errata tags set ProductComponentRelation"""
+    for e in released_errata_tags:
+        slow_load_errata.delay(e, force_process=force_process)
+
+
+def _check_and_load_brew_tags(
+    build_id: str, tags: list[str], build_type: str, softwarebuild: SoftwareBuild
+) -> int:
```

Comment:
Is below missing `return 0` / `return new_relations_count` lines, or should the return type hint get fixed?
```suggestion
) -> None:
```